cloud-info/ams-wrapper.sh

```diff
@@ -41,15 +41,13 @@ AUTO_CONFIG_PATH="$(mktemp -d)"
 export CHECKIN_SECRETS_FILE="$CHECKIN_SECRETS_PATH/secrets.yaml"
```

Comment:
Suggest passing secrets into the environment instead of via file

Reply:
I created an issue for this one: https://github.com/EGI-Federation/fedcloud-catchall-operations/issues/341